packages/blocks/src/_common/embed-block-helper/helper.ts

```diff
@@ -48,3 +54,73 @@ export function createEmbedBlock<
     view,
   };
 }
+
+// ========== Link Preview ==========
+
+export class LinkPreviewer {
```

Comment:
I'm not sure if it's appropriate to add the preview config with the common embed block helper @Saul-Mirone.

Also, would be good to clarify the updated API in PR description.

Reply:
pr's description has been updated